launch/sim_low_level_motion.launch

```diff
@@ -1,5 +1,5 @@
 <launch>
-    <node name="low_level_motion_controller" pkg="iarc7_motion" type="low_level_motion_controller">
+    <node name="low_level_motion_controller" pkg="iarc7_motion" type="low_level_motion_controller" output="screen">
```

Comment:
I really don't like having this in the launch file, it should be passed on the command line of you want the output displayed